src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1461,4 +1466,29 @@ public static boolean onFarmlandTrample(World world, BlockPos pos, IBlockState s
         return false;
     }
 
+    private static final Map<DataSerializer<?>, DataSerializerEntry> serializerEntries = GameData.getSerializerMap();
+    private static final ForgeRegistry<DataSerializerEntry> serializerRegistry = (ForgeRegistry<DataSerializerEntry>) ForgeRegistries.DATA_SERIALIZERS;
```

Comment:
Is there much reason to have this field?

Reply:
Not much - it does make the following code cleaner though.

There's something similar with `GameData.entityRegistry`, although that's used in patches, so it's more of an issue there.